ersilia/hub/content/catalog.py

```diff
@@ -28,19 +28,29 @@ class CatalogTable(object):
     def __init__(self, data, columns):
         self.data = data
         self.columns = columns
+    
+    def as_table(self):
+        column_widths = [
+            max(len(str(item)) if item is not None else 0 for item in [col] + [row[i] for row in self.data])
+            for i, col in enumerate(self.columns)
+        ]
+        row_format = " | ".join(f"{{:<{width}}}" for width in column_widths)
 
-    def as_list_of_dicts(self):
-        R = []
-        for r in self.data:
-            d = {}
-            for i, c in enumerate(self.columns):
-                d[c] = r[i]
-            R += [d]
-        return R
+        table = "┌" + "┬".join("─" * (width + 2) for width in column_widths) + "┐\n" 
+        table += "│ " + row_format.format(*self.columns) + " │\n"  
+        table += "├" + "┼".join("─" * (width + 2) for width in column_widths) + "┤\n"
+
+        for index, row in enumerate(self.data):
+            row = [str(item) if item is not None else "" for item in row]
+            table += "│ " + row_format.format(*row) + " │\n"
+
+            if index < len(self.data) - 1:
+                table += "├" + "┼".join("─" * (width + 2) for width in column_widths) + "┤\n"
```

Comment:
Can we make these border characters as importable constants in the defaults.py file? Ideally we should not have strings as much as possible in the code.

Reply:
Yes, and I agree. I'll implement this change and update the PR shortly

Reply:
@DhanshreeA the change has been implemented. border characters and are now set as constants in the defaults.py file